src/components/MultipleAvatars.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Image, Text, View} from 'react-native';
+import styles from '../styles/styles';
+
+const propTypes = {
+    // Array of avatar URL
+    avatars: PropTypes.arrayOf(PropTypes.any),
```

Comment:
Can we call this `avatarImageURLs`?

Also, it should be `PropTypes.arrayOf(PropTypes.string)`, right?

Reply:
Ok.